x-pack/plugins/security_solution/public/management/components/console/components/side_panel/side_panel_content_layout.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, ReactNode } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule } from '@elastic/eui';
+
+export interface SidePanelContentLayoutProps {
+  children: ReactNode;
+  headerContent?: ReactNode;
+}
+
+/**
+ * A layout component for displaying content in the right-side panel of the console
+ */
+export const SidePanelContentLayout = memo<SidePanelContentLayoutProps>(
+  ({ headerContent, children }) => {
+    return (
+      <EuiFlexGroup
+        direction="column"
+        responsive={false}
+        wrap={false}
+        className="eui-fullHeight"
+        gutterSize="none"
+      >
+        {headerContent && (
+          <>
+            <EuiFlexItem grow={false} className="layout-container">
+              {headerContent}
+            </EuiFlexItem>
+            <EuiHorizontalRule margin="none" />
+          </>
+        )}
+        <EuiFlexItem grow className="eui-scrollBar eui-yScroll layout-container">
+          <div>{children}</div>
```

Comment:
Is this `div` neccessary?

Reply:
if I remember correctly, yes. It breaks out of the `flex` item behaviour for its children

Reply:
Ah yes! I think in this case though it should work without it. I'll suggest you give it a try.

Reply:
Ok. I tried it and it does not work as expected. The content of this container will stretch to be 100% of the height of the `EuiFlexItem`, which has `grow` on it. So if I have multiple children inside, they will be adjusted to at least fill the entire height, which then causes something like this:

<img width="947" alt="image" src="https://user-images.githubusercontent.com/56442535/171024606-3cc4acdd-5126-46dd-b10d-3bce8c471022.png">

Since this side panel will be used for more than just the Help content and could hold content of different heights, we want the content to follow a normal element flow - they are displayed at the very top of the "body" of the side panel; taking up only enough space to display its content.

Reply:
I see. 👍